script/install.sh

```diff
@@ -277,6 +277,11 @@ parse_args() {
             dmd | dmd-* | gdc | gdc-* | ldc | ldc-*)
                 compiler=$1
                 ;;
+
+            *)
+                usage
+                fatal "Unrecognized command-line parameter: $1"
```

Comment:
Maybe first showing the error message first?

Reply:
No. I intentionally put it after the usage text. You're more likely to notice stuff at the bottom of your terminal window (near the prompt) than near the top.

Reply:
Fair enough, I wouldn't have minded anyways

Reply:
Indeed, it's weird that most tools do it vice versa.